cypress/integration/pages/onDemandRadio/testsForAMPOnly.js

```diff
@@ -0,0 +1,58 @@
+import appConfig from '../../../../src/server/utilities/serviceConfigs';
+import envConfig from '../../../support/config/envs';
+import getEmbedUrl from './helpers';
+
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRunForAMPOnly = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRunForAMPOnly to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfigForAMPOnly = ({
+  service,
+  pageType,
+  variant,
+}) =>
+  describe(`testsThatFollowSmokeTestConfigForAMPOnly for ${service} ${pageType}`, () => {
+    describe('Audio Player', () => {
+      const { lang } = appConfig[service][variant];
+      let embedUrl;
+      let override;
+
+      beforeEach(() => {
+        if (Cypress.env('APP_ENV') === 'test') {
```

Comment:
Or this could just be 
```
let override = '';

if (Cypress.env('APP_ENV') !== 'local') {
 override = '?renderer_env=live';
}
```

Reply:
This would also use the override on live, which we don't need to do?